specifications/ProposalForFeeChange.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Proposal for bridge service fees
+author: z-j-lin, vtleonardo
+discussion: <!--- Discussion URL -->
+team: Backend
+category: Development
+status: Draft
+related: <!-- Issue or Spec [URL] -->
```

Comment:
Is bridge pool or any other spec or existing files in the system related to this.